test/sample-test/sample_test_launcher.py

```diff
@@ -0,0 +1,262 @@
+    self._namespace = namespace
+    self._sample_test_result = 'junit_Sample%sOutput.xml' % self._test_name
+    self._sample_test_output = self._results_gcs_dir
+    self._work_dir = self.BASE_DIR + '/samples/core/' + self._test_name
+
+  def check_result(self):
+    os.chdir(self.TEST_DIR)
+    subprocess.call([
+        sys.executable,
+        'run_sample_test.py',
+        '--input',
+        '%s/%s.yaml' % (self._work_dir, self._test_name),
+        '--result',
+        self._sample_test_result,
+        '--output',
+        self._sample_test_output,
+        '--testname',
+        self._test_name,
+        '--namespace',
+        self._namespace
+    ])
+    print('Copy the test results to GCS %s/' % self._results_gcs_dir)
+    working_bucket = PROJECT_NAME
+
+    utils.upload_blob(
+        working_bucket,
+        self._sample_test_result,
+        self._results_gcs_dir + '/' + self._sample_test_result
+    )
+
+  def check_notebook_result(self):
+    # Workaround because papermill does not directly return exit code.
+    exit_code = '1' if PAPERMILL_ERR_MSG in \
+                     open('%s.ipynb' % self._test_name).read() else '0'
+
+    os.chdir(self.TEST_DIR)
+    if self._test_name == 'dsl_static_type_checking':
+      subprocess.call([
+          sys.executable,
+          'check_notebook_results.py',
+          '--testname',
+          self._test_name,
+          '--result',
+          self._sample_test_result,
+          '--exit-code',
+          exit_code
+      ])
+    else:
+      subprocess.call([
+          sys.executable,
+          'check_notebook_results.py',
+          '--testname',
+          self._test_name,
+          '--result',
+          self._sample_test_result,
+          '--namespace',
+          self._namespace,
+          '--exit-code',
+          exit_code
+      ])
+
+    print('Copy the test results to GCS %s/' % self._results_gcs_dir)
+    working_bucket = PROJECT_NAME
+
+    utils.upload_blob(
+        working_bucket,
+        self._sample_test_result,
+        self._results_gcs_dir + '/' + self._sample_test_result
+    )
+
+  def _run_test(self):
+    if len(self._results_gcs_dir) == 0:
+      return 1
+
+    os.chdir(self._work_dir)
+    print('Run the sample tests...')
+
+    # For presubmit check, do not do any image injection as for now.
+    # Notebook samples need to be papermilled first.
+    if self._test_name == 'lightweight_component':
+      pm.execute_notebook(
+          input_path='Lightweight Python components - basics.ipynb',
+          output_path='%s.ipynb' % self._test_name,
+          parameters=dict(
+              EXPERIMENT_NAME='%s-test' % self._test_name
+          )
+      )
+    elif self._test_name == 'dsl_static_type_checking':
+      pm.execute_notebook(
+          input_path='DSL Static Type Checking.ipynb',
+          output_path='%s.ipynb' % self._test_name,
+          parameters=dict(
+              KFP_PACKAGE='tmp/kfp.tar.gz',
```

Comment:
Looks like this sample does not require the KFP_PACKAGE any more.

Reply:
Agreed.